### DISCUSSION TITLE
Are all 4 banks of the DX7 patches in Minidexed?

### DISCUSSION THREAD

**Discussion by Alan-Angel** [probonopd/MiniDexed#661] [Q&A] [answered]
Hi, I feel like I've run across this issue before, I'm running Minidexed as a piece of outboard gear on a Pi using cubase on my Laptop. I have 8 discrete Midi channels set up and I can access the first 32 preset sounds of the DX7 using program changes in Cubase, but after that I just get init patches. Bank change meessages don't seem to do anything and it doesn't matter how I set up my ini file. Has anyone come accross this issue before? Do I need to install all my own syx files for stock DX7 patches?

**Comment by probonopd** [comment, marked as answer]
Yes, you need to install the syx files as per

```
#!/bin/sh

# Get voices from
# https://yamahablackboxes.com/collection/yamaha-dx7-synthesizer/patches/

mkdir -p sysex/voice/

DIR="https://yamahablackboxes.com/patches/dx7/factory"

wget -c "${DIR}"/rom3a.syx -O sysex/voice/000001_rom3a.syx
wget -c "${DIR}"/rom3b.syx -O sysex/voice/000002_rom3b.syx
wget -c "${DIR}"/rom4a.syx -O sysex/voice/000003_rom4a.syx
wget -c "${DIR}"/rom4b.syx -O sysex/voice/000004_rom4b.syx
```

**Reply by Alan-Angel** [reply]
Thanks. I thought as much, I can do that. So where are the first 32 patches sitting on the SD card currently? Are they tied to performance.ini or something?

**Reply by probonopd** [reply]
`performance.ini` can contain its own voice data.
